src/lib/db/seed.ts

```diff
@@ -106,15 +106,22 @@ const createDefaultEntities = async (db: Database, spaces: Space[], personas: Pe
 
 	for (const space of spaces) {
 		const componentName = findFirstComponentName(space.view);
+		if (componentName === 'Todo') {
+			await generateTodo(db, nextPersona().persona_id, space.space_id);
+		}
 		if (!componentName || !(componentName in entitiesContents)) {
 			continue;
 		}
 		const entityContents = entitiesContents[componentName];
 		for (const entityContent of entityContents) {
-			await db.repos.entity.create(nextPersona().persona_id, space.space_id, {
-				type: 'Note',
-				content: entityContent,
-			});
+			await db.repos.entity.create(
+				nextPersona().persona_id,
+				{
+					type: 'Note',
+					content: entityContent,
+				},
+				space.space_id,
+			);
```

Comment:
```suggestion
			unwrap(
				await db.repos.entity.create(
					nextPersona().persona_id,
					{
						type: 'Note',
						content: entityContent,
					},
					space.space_id,
				),
			);
```

Realized this will be slightly more robust if we wrap all seed service/db calls in `unwrap`, even if they don't use the return value.